### PR TITLE
wip: Link speaker photo and speaker or session name to view page

### LINKED ISSUE #6376: Organizer Session Overview Page and Speaker Overview Page: Link speaker photo and speaker or session name to view page
Please link the left hand rows for name or title on the following organizer pages to the session view or the speaker view page:
* The image/speaker name on the speaker overview page should link to the speaker view page
* The session title should link to the session view page

Speaker overview page
Screenshot from 2021-01-14 23-22-38

Session overview page
Screenshot from 2021-01-14 23-19-54

### PR DESCRIPTION
**Pull request by afzal442**
Fixes #6376 

#### Changes proposed in this pull request:

- Added the links required
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by afzal442** [comment]
@Creatoon collaborate me where I m missing or linking issue

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
i don't think this issue is talking about directing to public page of event.

**Comment by afzal442** [comment]
Thanks. So where to link? Plz suggest.

**Comment by afzal442** [comment]
@sachinchauhan2889 @iamareebjamal plz review this. Thanks.

File: app/templates/components/ui-table/cell/events/view/speakers/speaker-logo.hbs [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <div class="ui">
-  <img alt="Speaker" class="ui middle aligned tiny circular image" src="{{if this.record this.record (if this.extraRecords.photoUrl this.extraRecords.photoUrl '/images/placeholders/avatar.png')}}">
-</div>
\ No newline at end of file
+ <a class="image" href="{{href-to 'events.view.speaker.view'}}"><img alt="Speaker" class="ui middle aligned tiny circular image" src="{{if this.record this.record (if this.extraRecords.photoUrl this.extraRecords.photoUrl '/images/placeholders/avatar.png')}}"></a>
```
Comment (afzal442) [inline app/templates/components/ui-table/cell/events/view/speakers/speaker-logo.hbs:2]: href didn't work.

**Comment by iamareebjamal** [comment]
Do hit and trial on local machine, and make the PR WIP

**Comment by afzal442** [comment]
> Do hit and trial on local machine, and make the PR WIP

Alright Sir, I will. Thanks.

**Pull request closed without merging**